kitsune/search/v2/base.py

```diff
@@ -87,7 +87,7 @@ def to_action(self, action=None, **kwargs):
         # if we have a bulk update, we need to include the meta info
         # and return the data by calling the to_dict() method of DSL
         if is_bulk := kwargs.pop("is_bulk", False):
-            payload = self.to_dict(include_meta=is_bulk)
+            payload = self.to_dict(include_meta=is_bulk, skip_empty=False)
```

Comment:
Why do we need these empty values?

Reply:
Previously the tests would fail without this, after some changes that's no longer the case, so I added some more explicit tests in https://github.com/mozilla/kitsune/pull/4602